tests/bdd/features/lti_certification/section_4_levels_of_user_information.feature

```diff
@@ -5,41 +5,134 @@ Feature: Section 4 - Levels of User Information
   same resource_link_id (in a new context) but a different value for the
   user_id parameter. These tests only apply to LTI 1.
 
+  Background:
+    Given standard setup for LTI section 4
+
   @v1.0 @v1.1 @v1.2 @required
   Scenario: Test 4.1 - Launch as a learner with email but no name
     # Expected result: The tool should handle the lack of name information in
     # a user-friendly manner, which could be that they are denied access
 
+    Given the user is 'Sally'
+      And I set the fixture 'params' key 'roles' to 'Learner'
+      And I update the fixture 'params' with
+      | Key                              | Value            |
+      | lis_person_name_full             | *MISSING*        |
+      | lis_person_name_family           | *MISSING*        |
+      | lis_person_name_given            | *MISSING*        |
+
+     When I make an LTI launch request
+
+     Then the assigment opens successfully
+      But the user only has learner privileges
```

Comment:
The "But" and "only" in this wording makes it sound like this is a sad path, when in fact it's part of the happy path. In a later test, "And the user has instructor privileges" is used. Possible to use that step-wording here?

Reply:
And and But are interchangeable syntactic sugar, so we can do what we like!

Reply:
I've updated this now.